api/v1alpha1/envoygateway_types.go

```diff
@@ -225,6 +226,10 @@ type EnvoyGatewayKubernetesProvider struct {
 	// ShutdownManager defines the configuration for the shutdown manager.
 	// +optional
 	ShutdownManager *ShutdownManager `json:"shutdownManager,omitempty"`
+
+	// InitManager defines the configuration for the init manager.
```

Comment:
can we not expose this in the initial PR, and infer image name and pull policy from shutdown manager internally

Reply:
Sure, but could you elaborate on the reasoning?

Reply:
ideally dont want to confuse users with another setting they are unlikely to use, and this setting/implementation of using init containers will hopefully go away in the future

Reply:
Removed